packages/dendron-next-server/hooks/useApplyGraphConfig.tsx

```diff
@@ -0,0 +1,88 @@
+import cytoscape from "cytoscape";
+import { useEffect, useState } from "react";
+import { createLogger } from "../../common-frontend/lib";
```

Comment:
watch out for relative imports into workspace packages. see [here](https://github.com/dendronhq/dendron-site/blob/dev/vault/dendron.dev.troubleshooting.md#L12:L12)